R/new_functions.R

```diff
@@ -83,6 +83,10 @@ make_dictionary <- function(x) {
 #' @export
 load_experiment <- function(experiment, model, dir = "") {
 
+  if (!file.exists(model)) {
+    stop(paste0("There is no file \"", model, "\"."))
```

Comment:
already in show_experiment, you can remove it

Reply:
done